<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JPCTab</class>
 <widget class="QWidget" name="JPCTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="import_jpc">
       <property name="text">
        <string>Import JPC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_jpc">
       <property name="text">
        <string>Export JPC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_particles_from_folder">
       <property name="text">
        <string>Add Particles From Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_jpc_folder">
       <property name="text">
        <string>Export Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="jpc_particles_tree">
       <column>
        <property name="text">
         <string>Particle ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Chunk Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="jpc_sidebar" native="true">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="jpc_sidebar_label">
          <property name="text">
           <string>Extra information will be displayed here as necessary.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>298</width>
             <height>443</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionOpenJPCImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionReplaceJPCImage">
   <property name="text">
    <string>Replace Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
